metadataStore/commands.py

```diff
@@ -565,12 +565,12 @@ def find_last(num=1):
 
     Returns
     -------
-    begin_run_event: list
+    run_start: list
         Returns a list of the last ``num`` headers created.
         List of metadataStore.document.Document objects
         **NOTE**: DOES NOT RETURN THE EVENTS.
     """
-    br_objects = [br_obj for br_obj in BeginRunEvent.objects.order_by('-_id')[0:num]]
+    br_objects = [br_obj for br_obj in RunStart.objects.order_by('-_id')[0:num]]
```

Comment:
You're right. I will change this to conditional find(find_one for limit=1, and find for limit with N). That significantly increases query performance when pulling a single document.